src/java.base/share/man/java.1

```diff
@@ -4005,6 +4012,15 @@ Example:
 \f[CB]\-Xlog:gc:garbage\-collection.log\f[R]
 .RE
 .TP
+.B \f[CB]\-XX:+ExtendedDTraceProbes\f[R]
+\f[B]Linux and macOS:\f[R] Enables additional \f[CB]dtrace\f[R] tool probes
+that affect the performance.
```

Comment:
Existing grammatical nit: please delete 'the'.

Reply:
done